core/types/block.go

```diff
@@ -367,6 +367,16 @@ func (b *Block) Size() uint64 {
 	return uint64(c)
 }
 
+// PayloadSize returns the sum of all transactions in a block.
```

Comment:
How about fixing the comment to
"// PayloadSize returns the sum of size of all transactions in a block."?

Reply:
will change to 
// PayloadSize returns the sum of all transactions' sizes in a block.